environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/azure/AzureResourceGroupRequest.java

```diff
@@ -0,0 +1,59 @@
+package com.sequenceiq.environment.api.v1.environment.model.request.azure;
+
+import com.sequenceiq.environment.api.doc.environment.EnvironmentModelDescription;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(value = "AzureResourceGroupV1Parameters")
+public class AzureResourceGroupRequest {
+
+    @ApiModelProperty(EnvironmentModelDescription.EXISTING_RESOURCE_GROUP_NAME)
+    private String name;
+
+    @ApiModelProperty(EnvironmentModelDescription.USE_SINGLE_RESOURCE_GROUP)
+    private Boolean single;
```

Comment:
I guess the default value should be true

Reply:
can we rename something else ?

Reply:
> I guess the default value should be true

No, for now false. When the feature is ready then we can set it to true.